app/src/main/res/values/dimens.xml

```diff
@@ -29,5 +29,6 @@
     <dimen name="margin_very_small">2dp</dimen>
     <dimen name="margin_large">10dp</dimen>
     <dimen name="message_textview_bottom_padding">3dp</dimen>
+    <dimen name="textview_layout_max_width">240dp</dimen>
```

Comment:
It was suggested by @iamareebjamal in #653 that fixing value is not a solution here.